.github/workflows/maven.yml

```diff
@@ -15,9 +15,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
Why is there a long hash here? Shouldn't it just be "v4"?

Reply:
See my response on the other instance of `actions/checkout`